modules/mobile/docs/schemas/v1/ImmunizationsV1.yml

```diff
@@ -1,35 +1,37 @@
 type: object
+additionalAttributes: false
 required:
   - data
   - meta
 properties:
   data:
     type: array
     items:
-      type: object
       $ref: "../ImmunizationRecord.yml"
   meta:
     type: object
+    additionalAttributes: false
     required:
       - pagination
     properties:
       pagination:
         type: object
+        additionalAttributes: false
         required:
           - currentPage
           - perPage
           - totalPages
           - totalEntries
         properties:
           currentPage:
-            type: number
```

Comment:
Changed to integer because number includes both integers and floats. These can't be floats so I figured I should be more precise.